src/main/java/org/terasology/logic/health/DamageAuthoritySystem.java

```diff
@@ -198,8 +208,9 @@ public void onCrash(HorizontalCollisionEvent event, EntityRef entity, HealthComp
         Vector3f vel = new Vector3f(event.getVelocity());
         vel.y = 0;
         float speed = vel.length();
+        float height = heightOfCharacter(entity);
 
-        highSpeedDamage(speed, entity, health.horizontalDamageSpeedThreshold, health.excessSpeedDamageMultiplier);
+        highSpeedDamage(speed, entity, height < 30 ? health.horizontalDamageSpeedThreshold : 10000, health.excessSpeedDamageMultiplier);
```

Comment:
Should this be hardcoded? maybe this can be moved to some constant in health component.